gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,54 @@ def get_possible_database_path():
     return path
 
 
+def create_possible_database_path():
+    """Create the directory to what is possibly a GRASS Database.
+
+    Create directory named grassdata in the usual locations.   
+    
+    Returns the new path as a string.  
+    """
+    home = os.path.expanduser('~') 
+    tmp = os.path.join(tempfile.gettempdir(),
+                               "grassdata_{}".format(getpass.getuser()))           
+    
+    # Create independent "grassdata" in the home Linux dir
+    if sys.platform.startswith('linux'):
+        try:
+            path = os.path.join(home, "grassdata")
+            os.mkdir(path)
+        except OSError: 
+            print ("Creation of the directory {} failed".format(path))
+    
+    # Create independent "grassdata" in other dirs (Windows and Mac)
+    else:  
+        candidates = [
+            os.path.join(home, "Documents", "grassdata"),
+            os.path.join(home, "My Documents", "grassdata")
+        ]
+        
+        try:
+            # here goes everything which has potential unicode issues
+            candidates.append(os.path.join(home, _("Documents"), "grassdata"))
```

Comment:
Why _() in the path? Localized Windows should understand Documents.

Reply:
This comes from dac6d4add511cb47d3329d2c06fdb441b75822f4 - see the comment there. I would not bother with that for this function and just go with `Documents` only.

Reply:
As I said, here in this new function, I would not bother with translations here as this is more future oriented and we hope that every new system will be set in a way that there is "Documents". So, I'm saying, delete the `"candidates.append(os.path.join(home, _("Documents"), "grassdata"))"` part.